src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -150,9 +183,8 @@ def rewrite_jsvm_lcov(self, zip_file_path):
         out_dir = zip_file_path[:-4]
         out_file = out_dir + '.info'
 
-        zip_file = zipfile.ZipFile(zip_file_path, 'r')
-        zip_file.extractall(out_dir)
-        zip_file.close()
+        with ZipFile(zip_file_path, 'r') as z:
```

Comment:
No change here, just using the context manager

Reply:
Are we certain that the `with` context manager immediately calls `z.close()` when leaving the scope/context?

Reply:
Yes (https://github.com/python/cpython/blob/2.7/Lib/zipfile.py#L800), and `close` will be called no matter if the operations succeed or not, which is better than before (if extractall failed, we weren't closing the file).